storybook/src/components/scrollable/scrollable.decorator.ts

```diff
@@ -2,5 +2,5 @@ import { ScrollableDecorator } from "dso-toolkit";
 import { html, TemplateResult } from "lit-html";
 
 export const decorator: ScrollableDecorator<TemplateResult> = (story) => html`
-  <div id="scrollable-mock" style="background-color: #efefef; height: 750px; max-width: 500px">${story()}</div>
+  <div id="scrollable-mock" style="background-color: #efefef; height: 100vh; max-width: 500px">${story()}</div>
```

Comment:
Dit is gedaan zodat de volledige hoogte van het Storybook iframe wordt benut:

<img width="715" alt="image" src="https://github.com/user-attachments/assets/af15b138-8e77-4298-9486-148b7071431d" />

In de test is het andersom: Dat het grijze vlak doorsteekt door de hoogte van het venster waardoor je tijdens het ontwikkelen niet het gehele component in zicht hebt.